### PR TITLE
feat(regenerate): updates based on latest API definitions

### PR DESCRIPTION
**Pull request by mamoonraja**
### Release-3 2019

This PR generates the SDK based on the latest API definitions.

### CODE REVIEW

File: Scripts/Services/LanguageTranslator/V3/LanguageTranslatorService.cs [hunk 1]
```diff
@@ -646,28 +726,121 @@ public bool ListModels(Callback<TranslationModels> callback, string source = nul
...
+        /// <param name="target">Language code that specifies the target language for translation. (optional)</param>
+        /// <param name="documentId">To use a previously submitted document as the source for a new translation, enter
+        /// the `document_id` of the document. (optional)</param>
+        /// <returns><see cref="DocumentStatus" />DocumentStatus</returns>
+        public bool TranslateDocument(Callback<DocumentStatus> callback, System.IO.MemoryStream file, string filename,  string fileContentType = null, string modelId = null, string source = null, string target = null, string documentId = null)
+        {
+            if (callback == null)
+                throw new ArgumentNullException("`callback` is required for `TranslateDocument`");
+            if (file == null)
+                throw new ArgumentNullException("`file` is required for `TranslateDocument`");
```
Comment (mediumTaj) [inline Scripts/Services/LanguageTranslator/V3/LanguageTranslatorService.cs]: Can you add a check for `filename` here? I think we need to add support for adding the check in the generator.
```cs
if (string.IsNullOrEmpty(filename))
    throw new ArgumentNullException("`filename` is required for `TranslateDocument`");
```

File: Tests/LanguageTranslatorV3IntegrationTests.cs [hunk 2]
```diff
@@ -248,5 +251,106 @@ public IEnumerator TestDeleteModel()
...
+                {
+                    Log.Debug("LanguageTranslatorServiceV3IntegrationTests", "Delete Document result: {0}", response.Response);
+                    deleteResponse = response.Result;
+                    Assert.IsNull(deleteResponse);
+                    Assert.IsNull(error);
+                },
+                documentId: documentId
+            );
+
+            yield return null;
```
Comment (mediumTaj) [inline Tests/LanguageTranslatorV3IntegrationTests.cs]: Is this hanging here? you prob want 
```cs
while (deleteResponse == null)
    yield return null;
```

File: Tests/LanguageTranslatorV3IntegrationTests.cs [hunk 3]
```diff
@@ -248,5 +251,106 @@ public IEnumerator TestDeleteModel()
...
+        {
+            Log.Debug("LanguageTranslatorServiceV3IntegrationTests", "Deleteing the Document...");
+            object deleteResponse = null;
+
+            service.DeleteDocument(
+                callback: (DetailedResponse<object> response, IBMError error) =>
+                {
+                    Log.Debug("LanguageTranslatorServiceV3IntegrationTests", "Delete Document result: {0}", response.Response);
+                    deleteResponse = response.Result;
+                    Assert.IsNull(deleteResponse);
```
Comment (mediumTaj) [inline Tests/LanguageTranslatorV3IntegrationTests.cs]: You can also check
```cs
Assert.IsTrue(response.StatusCode == 204);
```

Comment (mamoonraja) [reply Tests/LanguageTranslatorV3IntegrationTests.cs]: That makes more sense :+1:

**Pull request merged**